code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1581,6 +1584,22 @@
 	..()
 	reagents.clear_reagents()
 
+/obj/item/weapon/reagent_containers/food/snacks/pie/clovercreampie
+	name = "whipped clover pie"
+	desc = "Traditional dish in the Clownplanet's Irish exclusion zone."
+	icon_state = "clovercreampie"
+	trash = /obj/item/trash/pietin
+	food_flags = FOOD_SWEET
+
+/obj/item/weapon/reagent_containers/food/snacks/pie/clovercreampie/New()
+	..()
+	bitesize = 3
+	if(prob(25))
+		reagents.add_reagent(NUTRIMENT, 10) //Lucky pie is more nutritious
```

Comment:
10 bonus is quite a large amount don't you think? common meals are 4-6 and even wish soup getting lucky is 8 (and it has 0 nutriment if it fails)

Reply:
It kinda is, but the main purpose of the clover pies would be to throw, so having 10 nutriment instead of just 5 seems fine. I can lower it a bit if you want though.

Reply:
Why not make the chance based on the luckiness of the completed dish instead of 25%, at least?

Reply:
I don't really understand luck code and it feels overly complex for what can be achieved with a 1/4 chance. I'll reduce the nutriment instead, down to 8.

Reply:
You'd have to call `chefmob.lucky_prob(25)` in place of `prob(25)` to have it affected by the chef's luck. The other arguments for `lucky_prob()` aren't strictly necessary but are just for tuning how much luck affects it.